csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/InferenceTest.cs

```diff
@@ -146,6 +152,10 @@ public void TestSessionOptions()
                 opt.AppendExecutionProvider_Nnapi(0);
 #endif
 
+#if USE_TVM
+                opt.AppendExecutionProvider_Tvm("Vulkan -device=amd_apu");
```

Comment:
I think the TVM EP was removed: https://github.com/microsoft/onnxruntime/pull/22827